tests/query_runner/test_influx_db.py

```diff
@@ -0,0 +1,58 @@
+import json
+
+from influxdb.resultset import ResultSet
+
+from redash.query_runner import (
+    TYPE_FLOAT,
+    TYPE_INTEGER,
+    TYPE_STRING,
+)
+from redash.query_runner.influx_db import _transform_result
+
+raw = {
+    "series": [
+        {
+            "name": "typetest",
+            "columns": ["time", "k1", "v1", "v2"],
+            "values": [
+                ["2023-10-06T13:30:51.323358136Z", "foo", 0.5, 2],
+                ["2023-10-06T13:31:08.882953339Z", "bar", 0.6, 4],
+            ],
+        }
+    ]
+}
+
+raw_no_rows = {"series": [{"name": "typetest", "columns": ["time", "k1", "v1", "v2"], "values": []}]}
+
+
+def test_result():
```

Comment:
nit: can we rename the tests to be more descriptive? Maybe something like `test_influxdb_result_types_with_rows` and `test_influxdb_result_types_with_no_rows_are_string`?